integration-tests/requirements.txt

```diff
@@ -4,14 +4,19 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
+# To install pip-compile, run:
+#    pip install pip-tools
+#
 behave==1.2.5
 docker-py==1.10.6
 docker-pycreds==0.2.1     # via docker-py
+enum34==1.1.6             # via parse-type
 jsonschema==2.5.1
 parse-type==0.3.4         # via behave
 parse==1.6.6              # via behave, parse-type
 py==1.4.31                # via pytest
 pytest==3.0.3
+repoze.lru==0.6           # via jsonschema
```

Comment:
Yes please. You can create one easily by pressing `Revert`button next to the *pkajaba merged commit c1586c4 into fabric8-analytics:master* message.